auth-source-pass.el

```diff
@@ -272,16 +272,34 @@ If ENTRIES is nil, use the result of calling `auth-source-pass-entries' instead.
                entry))
    (or entries (auth-source-pass-entries))))
 
-(defun auth-source-pass--generate-entry-suffixes (hostname user port)
+(defun auth-source-pass--generate-entry-suffixes (hostname users ports)
   "Return a list of possible entry path suffixes in the password-store.
 
 Based on the supported filename patterns for HOSTNAME, USER, &
```

Comment:
please change the variable names here to match the name of the function parameters.

You may want to activate flycheck and checkdoc to see the problems within the docstrings. You may also want to run `make check`.